src/utils/axios.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+export const getInfo = field=> {
```

Comment:
Separate the search into students search and tutors search. Follow the example of this
https://github.com/Joey-z-rp/jr-lms-demo/tree/master/src/utils/api